### PR TITLE
feat(CheckboxList): Added checkbox list component

### LINKED ISSUE #50: Create a Checkbox list component
Skjermbilde 2020-09-20 kl  15 06 53

### PR DESCRIPTION
**Pull request by TanyaDolgopolova**
Close #50

### CODE REVIEW

File: src/components/CheckboxList/CheckboxList.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <Flex sx={{
+      flexWrap: 'wrap',
+      justifyContent: 'flex-start'
+    }}>
+      {props.options.map((option) => {
+        return (
+          <Label>
+            <Checkbox
+              id={option.id}
```
Comment (cobraz) [inline src/components/CheckboxList/CheckboxList.tsx:18]: Shouldn't this have a `key` value?

**Review by cobraz — CHANGES_REQUESTED** [review]

File: src/components/CheckboxList/CheckboxList.tsx (hunk 1 above)
Comment (TanyaDolgopolova) [reply src/components/CheckboxList/CheckboxList.tsx:18]: You are right, I will change it

**Review by cobraz — APPROVED** [review]
LGTM 🎉

File: src/components/CheckboxList/CheckboxList.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+              key={option.id}
```
Comment (kvokov) [inline src/components/CheckboxList/CheckboxList.tsx:19]: key should be specified for `<Label>` as root element there

**Review by kvokov — APPROVED** [review]

**Pull request merged**